03_API_Reference/01_Ethereum_Provider.md

```diff
@@ -105,13 +259,45 @@ ethereum.send({
  // Like a typical promise, returns an error on rejection.
 })
 ```
-### ethereum.sendAsync(options, callback)
 
-_To be superceded by the promise-returning send() method in [EIP 1193](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1193.md)._
+### ethereum.on(eventName, callback)
+
+The provider supports listening for all events specified in [EIP 1193](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1193.md#events).
 
-Sends a message to the web3 browser. Message format maps to the format of [the Ethereum JSON-RPC API](https://github.com/ethereum/wiki/wiki/JSON-RPC#json-rpc-methods).
+The following are especially important for managing the state of your dapp:
+- `accountsChanged`, returns an array of the currently available accounts.
+- `chainChanged`, returns the hex-formatted chain ID string of the currently used chain/network.
+- `networkChanged`, _(Discouraged)_ returns decimal-formatted network ID string of the currently used chain/network.
 
-Here's an example of everyone's favorite method, sending a transaction, which is both how Ether is sent, and how smart contract methods are called:
+#### Example
+```javascript
+ethereum.on('accountsChanged', function (accounts) {
+  // Time to reload your interface with accounts[0]!
+})
+
+ethereum.on('chainChanged', function (chainId) {
+  // Time to make sure your any calls are directed to the correct chain!
+})
+```
+
+#### List of Chain and Network IDs
+```javascript
+'1': Ethereum Main Network
+'2': Morden Test network
+'3': Ropsten Test Network
+'4': Rinkeby Test Network
+'5': Goerli Test Network
+'42': Kovan Test Network
+```
+
+### ethereum.sendAsync(options, callback) (Deprecated)
+
+_We strongly discourage the use of this method, which may be removed in the future._
```

Comment:
Ideally we would link to the recommended alternative for each deprecated item. Maybe a line following each of these "We strongly discourage" sentences would be the right place for that?